util/db_info_dumper.cc

```diff
@@ -118,7 +118,7 @@ void DumpDBFileSummary(const DBOptions& options, const std::string& dbname) {
           char str[8];
           snprintf(str, sizeof(str), "%" PRIu64, file_size);
           wal_info.append(file).append(" size: ").
-              append(str, sizeof(str)).append(" ;");
+              append(str).append(" ; ");
```

Comment:
snprintf() ensures that the str will be terminated with a null character, so no risk here :)